Buildings/Examples/VAVReheat/Guideline36.mo

```diff
@@ -17,27 +19,37 @@ model Guideline36
   Buildings.Controls.OBC.ASHRAE.G36_PR1.TerminalUnits.Controller conVAVCor(
     V_flow_nominal=mCor_flow_nominal/1.2,
     AFlo=AFloCor,
-    final samplePeriod=samplePeriod) "Controller for terminal unit corridor"
+    final samplePeriod=samplePeriod,
+    final VDisSetMin_flow=max(1.5*VCorOA_flow_nominal, 0.15*mCor_flow_nominal/1.2))
```

Comment:
What is the benefit of having this `max()`, rather than the default `0.15*V_flow_nominal` (which is `0.15*mCor_flow_nominal/1.2)`?

Reply:
This is per ASHRAE Standard 62.1-2019 Eq. (6-9): `VMin = 1.5 * Voz`. However, this equation is part of the simplified procedure (and can lead to values as low as `0.1*V_flow_nominal`) so I chose the compromise of the max with the original default setting which was `0.15*V_flow_nominal` (which offers a generic, not project specific, order of magnitude but can create critical zones).
The requirement from G36 is

> For ASHRAE Standard 62.1 ventilation, select Vmin to prevent creating critical zones.

Taking the max will not create critical zones. It can only lower the zone primary outdoor air fraction Zpz so it will not decrease the system ventilation efficiency.